cli/docs/cli.json

```diff
@@ -357,12 +587,74 @@
               "long": "disk",
               "help": "Name or ID of the disk"
             },
+            {
+              "long": "help",
+              "short": "h",
+              "help": "Print help"
+            },
             {
               "long": "project",
               "help": "Name or ID of the project"
             }
           ]
         },
+        {
+          "name": "help",
+          "about": "Print this message or the help of the given subcommand(s)",
+          "subcommands": [
+            {
+              "name": "create",
+              "about": "Create a disk"
+            },
+            {
+              "name": "delete",
+              "about": "Delete a disk"
+            },
+            {
+              "name": "help",
+              "about": "Print this message or the help of the given subcommand(s)"
+            },
```

Comment:
uh... is this correct? I suppose it technically is:

```
$ oxide help vpc help
Print this message or the help of the given subcommand(s)

Usage: oxide vpc help [COMMAND]...

Arguments:
  [COMMAND]...  Print help for the subcommand(s)
```

But why is that possible at all?

Reply:
It seems to be correct. Perhaps its not helpful to include in this output based on how the output is used. What do you think?

I think that invoking `build()` is "correct" based on what we're doing:
> Prepare for introspecting on all included [Command](https://docs.rs/clap/latest/clap/struct.Command.html)s
> 
> Call this on the top-level [Command](https://docs.rs/clap/latest/clap/struct.Command.html) when done building and before reading state for cases like completions, custom help output, etc.

Further, if we don't call `build`, as of 4.4.14 we get extraneous true/false values for boolean flags.

Reply:
I think I want to see how it feels in the docs, because that’s all this JSON is for. It feels noisy here, I think it will feel noisy there. Or maybe we could be clever and do something on the display side to collapse it under a “meta commands” (or soemthing) heading on every page.

Reply:
Agreed. I don't think we should include any of the help stuff in this json output.